generators/entity-client/templates/angular/src/main/webapp/app/entities/entity-management-routing.module.ts.ejs

```diff
@@ -88,6 +88,11 @@ const <%= entityInstance %>Route: Routes = [
         RouterModule.forChild(<%= entityInstance %>Route),
         <%= microservicePrefix %><%= entityAngularName %>Module,
     ],
+    <%_ if (!readOnly) { _%>
+    declarations: [
+        <%= entityAngularName %>RouteUpdateComponent,
+    ],
+    <%_ } _%>
```

Comment:
```suggestion
```
Don't declare components in routing module. Use entity module instead.

Reply:
This was a leftover of a different approach .
Done